backend/vaa-strapi/.env.example

```diff
@@ -3,19 +3,19 @@
 ## Use the .env file in the project root when using Docker
 
 # Backend dependencies
-STRAPI_HOST=0.0.0.0
+STRAPI_HOST=localhost
```

Comment:
This needs to be `localhost` in order for the Docker service to be accessible from outside its own Docker compose stack (which is exactly what we need if we spin up only the backend service with this local `.env` file).